apps/condo/domains/organization/access/Organization.js

```diff
@@ -19,7 +19,12 @@ async function canReadOrganizations ({ authentication: { item: user }, context }
         if (residents.length === 0) {
             return false
         }
-        const organizations = compact(residents.map(resident => get(resident, ['organization', 'id'])))
+        const residentOrganizations = compact(residents.map(resident => get(resident, ['organization', 'id'])))
+
+        const residentsServiceConsumers = await ServiceConsumerServerUtils.getAll(context, { resident: { id_in: residents.map(resident => resident.id) } })
```

Comment:
deletedAt

Reply:
and for Residents

Reply:
and for user I guess
`if (user.deletedAt) return false`

Reply:
And what about field accesses?

Seems like there's nothing to hide in there for now....But there's already access for acquiring integration + resident + serviceConsumer....

May be we should add TODO to think about field-accesses later?

Reply:
Its an open question really, didnt get what you want